in_situ/swe/swe_decom.py

```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
+from space_packet_parser import xtcedef, parser
+import bitstring
+
+packet_file = Path('science_block_20221116_163611Z_idle.bin')
+with packet_file.open(mode='rb') as fh:
+    binary_data = bitstring.ConstBitStream(fh)
```

Comment:
I think you can ignore bitstring details according to the space_packet_parser docs: https://space-packet-parser.readthedocs.io/en/latest/users.html#examples

```suggestion
packet_file = Path('science_block_20221116_163611Z_idle.bin')
with packet_file.open(mode='rb') as binary_data:
    packet_generator = my_parser.generator(binary_data)
```

```python
with packet_file.open() as binary_data:
    packet_generator = my_parser.generator(binary_data)

    for packet in packet_generator:
        # Do something with the packet data
        print(packet.header['PKT_APID'])
        print(packet.data)
```